pkl-codegen-kotlin/src/main/kotlin/org/pkl/codegen/kotlin/KotlinCodeGenerator.kt

```diff
@@ -344,24 +338,25 @@ class KotlinCodeGenerator(
       return builder.build()
     }
 
+    // produce same output as default toString() method of data classes
     fun generateToStringMethod(): FunSpec {
       return FunSpec.builder("toString")
         .addModifiers(KModifier.OVERRIDE)
         .returns(STRING)
         .addStatement(
           "return %P",
-          buildString {
-            append(pClass.simpleName)
-            append("(")
-            var isFirst = true
-            for (propertyName in allProperties.keys) {
-              if (isFirst) isFirst = false else append(", ")
-              append(propertyName)
-              append("=$")
-              append(propertyName)
+          CodeBlock.builder()
+            .apply {
+              add("%L", pClass.toKotlinPoetName().simpleName)
```

Comment:
added `toKotlinPoetName()` here, which I think is necessary

Reply:
Ah, yup; good catch